distributed/client.py

```diff
@@ -1191,13 +1183,16 @@ def submit(self, func, *args, **kwargs):
             raise TypeError("First input to submit must be a callable function")
 
         key = kwargs.pop('key', None)
-        pure = kwargs.pop('pure', True)
         workers = kwargs.pop('workers', None)
         resources = kwargs.pop('resources', None)
         retries = kwargs.pop('retries', None)
         priority = kwargs.pop('priority', 0)
         fifo_timeout = kwargs.pop('fifo_timeout', '100ms')
         allow_other_workers = kwargs.pop('allow_other_workers', False)
+        actor = kwargs.pop('actor', False)
+        actors = kwargs.pop('actors', False)
+        actor = actor or actors
```

Comment:
So `actors` would be general purpose, but it seemed a bit odd for submit

    client.submit(Foo, actors=True)

It also seemed error prone to have one keyword for submit and one for map, so I just used both in both places.  I agree that this is wonky though and am not surprised that it would not survive review.  Do you have any suggestions?  Use `actors=` everywhere, including submit?